resources/packs/aws/aws_ec2.go

```diff
@@ -1127,6 +1127,46 @@ func (p *mqlAwsEc2Volume) init(args *resources.Args) (*resources.Args, AwsEc2Vol
 	return nil, nil, errors.New("volume does not exist")
 }
 
+func (d *mqlAwsEc2Instance) init(args *resources.Args) (*resources.Args, AwsEc2Instance, error) {
+	if len(*args) > 2 {
+		return args, nil, nil
+	}
+
+	if len(*args) == 0 {
+		if ids := getAssetIdentifier(d.MqlResource().MotorRuntime); ids != nil {
+			(*args)["arn"] = ids.arn
```

Comment:
Or, from another perspective to the `> 1`above: Is it intended, that the EC2 instance doesn't have the name here, as the other resources do?

Reply:
ya, name isn't stable for this resource, so we're just looking at the arn